webapp/channels/src/components/edit_post/edit_post.tsx

```diff
@@ -390,8 +391,11 @@ const EditPost = ({editingPost, actions, canEditPost, config, channelId, draft,
     };
 
     const handleEmojiClick = (emoji?: Emoji) => {
-        const emojiAlias = emoji && (((emoji as SystemEmoji).short_names && (emoji as SystemEmoji).short_names[0]) || emoji.name);
+        if (!emoji) {
+            return;
+        }
 
+        const emojiAlias = getEmojiName(emoji);
```

Comment:
I like having it be non-nullable since it seems to be used in most places where we know that the emoji is always defined. I don't actually know why this function takes an optional parameter since I can't think of why the EmojiPicker would ever call its callback without an emoji defined